cvmfs/cvmfs_server

```diff
@@ -899,7 +899,7 @@ close_transaction() {
     cvmfs_suid_helper rdonly_umount $name
   fi
   cvmfs_suid_helper clear_scratch $name
-  rm -rf "${tmp_dir}/*"
+  rm -rf ${tmp_dir}/*
```

Comment:
I believe, any spaces in paths or repo names or anything might surprise us anyways. ;-) The quoted thing doesn't work, because it intercepts the shell globbing. Probably something like `rm -rf "${tmp_dir}"/*` would work as expected...? Shell is evil...